includes/Experiments.php

```diff
@@ -355,14 +355,25 @@ public function get_experiments(): array {
 			/**
 			 * Author: @miina
 			 * Issue #7986
-			 * Creation date: 2021-08-08
+			 * Creation date: 2021-07-08
 			 */
 			[
 				'name'        => 'enableSmartTextColor',
 				'label'       => __( 'Smart text color', 'web-stories' ),
 				'description' => __( 'Enable text insertion with smart color ensuring good contrast with the background', 'web-stories' ),
 				'group'       => 'editor',
 			],
+			/**
+			 * Author: @merapi
+			 * Issue #7995
+			 * Creation date: 2021-08-13
+			 */
+			[
+				'name'        => 'enableSmartTextSetsColor',
```

Comment:
Is this flag actually used anywhere? Isn't the feature always enabled in the code currently?

Reply:
Create a flag - good.
Actually use the flag - even better 😄 .
Thanks for catching that!